compiler/src/parsing/parser.dyp

```diff
@@ -163,7 +163,7 @@ pattern :
   | const { Pat.constant ~loc:(symbol_rloc dyp) $1 }
   /* If the pattern uses an external ID, we know it's a constructor, not a variable */
   | ext_constructor { Pat.construct ~loc:(symbol_rloc dyp) $1 [] }
-  | [ID | special_id | primitive] { Pat.var ~loc:(symbol_rloc dyp) (mkstr dyp $1) }
+  | [ID | special_id | primitive] { Pat.var ~loc:(symbol_rloc dyp) (mkstr (symbol_rloc dyp) $1) }
```

Comment:
Can we merge this as a separate PR? I believe it was based on one of @ospencer's PRs?

Reply:
It was, but we decided to merge it with the formatter as it only impacts the formatter.